pkg/core/managers/apis/mesh/mesh_manager_test.go

```diff
@@ -84,7 +87,7 @@ var _ = Describe("Mesh Manager", func() {
 			Expect(err).ToNot(HaveOccurred())
 		})
 
-		It("should create default traffic permission", func() {
+		It("should create default resources", func() {
```

Comment:
Why? I add similar for the default route in my PR.

Reply:
because I also created a signing key here. I thought it's better to add a new and then creating a completely new test just for it.

Reply:
Makes sense, let's go with this more generic approach.